draft-ietf-scone-protocol.md

```diff
@@ -506,8 +543,80 @@ if is_long and (packet_version == SCONE1_VERSION or
 Once the throughput advice signal is updated,
 the network element updates the UDP checksum for the datagram.
 
-
-## Flows That Exceed Throughput Advicea
+A network element needs to ensure that it sends updated rate signals
+with no more than a monitoring period ({{time}}) between each.
+Because this depends on the availability of SCONE packets
+and packet loss can cause signals to be missed,
+network elements might need to update more often.
+
+Senders that send a SCONE packet
+or network elements that update SCONE packets
+every 20&ndash;30 seconds is likely sufficient to ensure that throughput advice is not lost.
+Senders can avoid synchronization of SCONE packets,
+which might cause network elements to miss updates,
```

Comment:
I'm not sure why synchronization might cause missed updates, but maybe it doesn't matter.